renderer/viewer/three/cameraShake.ts

```
@@ -0,0 +1,98 @@
+import * as THREE from 'three'
+import * as tweenJs from '@tweenjs/tween.js'
+
+export class CameraShake {
+  // TODO: temporarily disabled roll animation functionality
+  // eslint-disable-next-line @typescript-eslint/prefer-readonly
```

ESLint

[error] '@typescript-eslint/prefer-readonly' rule is disabled but never reported.

(eslint-comments/no-unused-disable)